### PR TITLE
SBERDOMA-1154 fix sender admin value

### PR DESCRIPTION
**Pull request by MrFoxPro**

### CODE REVIEW

**Review by dkoviazin — APPROVED** [review]

File: apps/condo/domains/common/utils/validation.utils.js [hunk 1]
```diff
@@ -19,33 +19,48 @@ function hasDbFields (databaseRequired, resolvedData, existingItem, context, add
...
-                    }
-                    else
-                    { 
-                        resolvedData[field] = cookies[field]
-                        continue
-                    }
-                    
+function hasDvAndSenderFields (resolvedData, context, addFieldValidationError) {
+    let hasDvField = true
+    if (!resolvedData.hasOwnProperty('dv')) {
```
Comment (toplenboren) [inline apps/condo/domains/common/utils/validation.utils.js:24]: I believe we would need a large confluence doc about the `dv` field in our system and how it works

- cookies,
- models

all that stuff

File: apps/condo/domains/billing/schema/BillingReceipt.js [hunk 2]
```diff
@@ -118,7 +118,7 @@ const BillingReceipt = new GQLListSchema('BillingReceipt', {
     },
     hooks: {
         validateInput: ({ resolvedData, existingItem, context, addValidationError }) => {
-            if (!hasRequestFields(['dv', 'sender'], resolvedData, context, addValidationError)) return
+            if (!hasDvAndSenderFields( resolvedData, context, addValidationError)) return
```
Comment (toplenboren) [inline apps/condo/domains/billing/schema/BillingReceipt.js:121]: Am I right, by saying that right now we would always need to set `hasDvAndSenderFields` in validation for our feature models? 

Maybe add it to codegen?

**Review by toplenboren — APPROVED** [review]

File: apps/condo/domains/billing/schema/BillingReceipt.js (hunk 2 above)
Comment (pahaz) [reply apps/condo/domains/billing/schema/BillingReceipt.js:121]: It will be nice!

File: apps/condo/index.js [hunk 3]
```diff
@@ -157,8 +158,25 @@ module.exports = {
...
+                true,
+                1,
+                {
+                    fingerprint: {
+                        presence: true,
+                        format: /^[a-zA-Z0-9!#$%()*+-;=,:[\]/.?@^_`{|}~]{5,42}$/,
+                        length: { minimum: 5, maximum: 42 },
+                    },
+                })
+            if (!isSenderValid) {
```
Comment (pahaz) [inline apps/condo/index.js:178]: is it ok? if not valid?

Comment (MrFoxPro) [reply apps/condo/index.js:178]: Yes

File: apps/condo/index.js [hunk 4]
```diff
@@ -157,8 +158,25 @@ module.exports = {
         app.use('/admin/', (req, res, next) => {
             if (req.url === '/api') return next()
             const cookies = nextCookie({ req })
-            if (!cookies.sender){
-                res.cookie('sender', makeId(12))
+            let isSenderValid = true
+
+            isSenderValid = hasValidJsonStructure(
+                {
+                    resolvedData: { sender: cookies['sender'] },
```
Comment (pahaz) [inline apps/condo/index.js:165]: dv?

**Pull request merged**